docs/wiki/Whats-new.md

```diff
@@ -43,6 +45,74 @@ This article will be updated as and when changes are made to the above and anyth
 
 Here's what's changed in Enterprise Scale/Azure Landing Zones:
 
+### 🆕 AMA Updates
```

Comment:
Same here for @arjenhuitema, do we also need to include a list of assignments that ALZ Portal/ARM/Bicep customers need to manually remove so they do not have duplicate assignments if they used the previous policies?

Can we just provide a bullet point list for the policies and assignment scopes they need to remove manually?